data/map.txt

```diff
@@ -27093,22 +27093,40 @@ planet "Elek Kartanu"
 
 planet "Ember Graveyard"
 	attributes "requires: quantum keystone"
-	description ""
+
+wormhole "Ember Graveyard"
+	linked
```

Comment:
The `linked` keyword doesn't sound right to me; every wormhole is linked to somewhere (otherwise it wouldn't be a wormhole). Maybe `mappable` (as in "explored and able to be shown on a map") would be a better keyword, since it controls if the link is shown on a map or not.